From 83d412cf08d76fab5e79b6f261c3145372337d5e Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Thu, 10 Jan 2019 11:38:38 -0500
Subject: [PATCH] preferences: disable pEpAutoDownload by default

Distributions should not automatically download this untrusted code;
should be opt-in by users or be packaged separately.
---
 package/prefs/defaultPrefs.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/prefs/defaultPrefs.js b/package/prefs/defaultPrefs.js
index d5ca1249..f8c10a47 100755
--- a/package/prefs/defaultPrefs.js
+++ b/package/prefs/defaultPrefs.js
@@ -204,7 +204,7 @@ pref("extensions.enigmail.warnDownloadContactKeys", true);
 pref("extensions.enigmail.wrapHtmlBeforeSend", true);
 
 // automatically download pepmda if it is available (without askin user)
-pref("extensions.enigmail.pEpAutoDownload", true);
+pref("extensions.enigmail.pEpAutoDownload", false);
 
 // holds the last result of the last check for pEp updates
 pref("extensions.enigmail.pEpLastUpdate", 0);
-- 
2.20.1

